OpenTabletDriver/Devices/WinUSB/WinUSBRootHub.cs

```diff
@@ -0,0 +1,133 @@
+using System;
+using System.Collections.Generic;
+using System.Runtime.ConstrainedExecution;
+using System.Runtime.InteropServices;
+using OpenTabletDriver.Native.Windows.CM;
+using OpenTabletDriver.Native.Windows.SetupApiStructs;
+using OpenTabletDriver.Native.Windows.USB;
+using OpenTabletDriver.Plugin;
+using OpenTabletDriver.Plugin.Attributes;
+using OpenTabletDriver.Plugin.Devices;
+using static OpenTabletDriver.Native.Windows.CfgMgr32;
+using static OpenTabletDriver.Native.Windows.SetupAPI;
+
+namespace OpenTabletDriver.Devices.WinUSB
+{
+    [RootHub, SupportedPlatform(PluginPlatform.Windows)]
+    public class WinUSBRootHub : CriticalFinalizerObject, IRootHub
+    {
+        private static readonly Guid[] _winUsbGuids = new Guid[]
+        {
+            // Standard WinUSB (Zadig)
+            new("{dee824ef-729b-4a0e-9c14-b7117d33a817}"),
+
+            // Huion WinUSB
+            new("{62f12d4c-3431-4efd-8dd7-8e9aab18d30c}"),
```

Comment:
I'm not a huge fan of `new()` without a type name unless its a really long type name.
```suggestion
            // Standard WinUSB (Zadig)
            new Guid("{dee824ef-729b-4a0e-9c14-b7117d33a817}"),
            // Huion WinUSB
            new Guid("{62f12d4c-3431-4efd-8dd7-8e9aab18d30c}"),
```